src/GTP.h

```diff
@@ -32,6 +32,10 @@ extern bool cfg_allow_pondering;
 extern int cfg_num_threads;
 extern int cfg_max_playouts;
 extern int cfg_max_visits;
+enum timemanage_t : int {
```

Comment:
I'd move the definition of timemanage_t elsewhere. UCTSearch.h perhaps?

Reply:
done